src/main/java/net/wurstclient/hacks/FeedAuraHack.java

```diff
@@ -107,7 +114,10 @@ public void onUpdate()
 			.filter(e -> !e.removed).filter(e -> e instanceof AnimalEntity)
 			.map(e -> (AnimalEntity)e).filter(e -> e.getHealth() > 0)
 			.filter(e -> player.squaredDistanceTo(e) <= rangeSq)
-			.filter(e -> e.isBreedingItem(heldStack)).filter(e -> e.canEat());
+			.filter(e -> e.isBreedingItem(heldStack)).filter(AnimalEntity::canEat);
+
+		if (filterBabies.isChecked())
+			stream = stream.filter(e -> !(e != null && e.isBaby()));

```

Comment:
Why would `e` ever be `null` here? It seems like that would cause problems before this check even runs.